src/util/KnowledgeGraphUtils.js

```diff
@@ -327,6 +327,21 @@ export default class KnowledgeGraphUtils {
     return result;
   }
 
+  //eslint-disable-next-line
```

Comment:
Why is the eslint-disable needed?

Reply:
Because this function doesn't use the `this` keyword

Reply:
Lint was complaining about the function not using `this` keyword but I changed it to static as it can be useful in other cases too. Removed the lint disable comment.